### PR TITLE
OM-572 | Admin UI enhancements

### PR DESCRIPTION
**Pull request by tuomas777**

### CODE REVIEW

File: users/admin.py [hunk 1]
```diff
@@ -8,9 +11,15 @@
 
 
 @admin.register(User)
-class UserAdminInline(VersionAdmin):
+class UserAdminInline(VersionAdmin, DjangoUserAdmin):
     inlines = [ProfileAdminInline]
 
+    def get_fieldsets(self, request, obj=None):
+        fieldsets = deepcopy(super().get_fieldsets(request, obj))
+        fieldsets[1][1]["fields"] += ("uuid",)
```
Comment (pjlampil) [inline users/admin.py]: I get this while trying to create a new user:

```
Exception Type: | IndexError
-- | --
tuple index out of range
/app/users/admin.py in get_fieldsets, line 19
```

Comment (tuomas777) [reply users/admin.py]: Yeah, apparently forgot to test user creation with the newest iteration of the thing. Should be fixed now.

**Comment by pjlampil** [comment]
Looks good but how about merging commits to reduce code churn?

**Comment by tuomas777** [comment]
> Looks good but how about merging commits to reduce code churn?

Yup that was the idea, could have been more verbal about that :)

**Pull request merged**